File: Pratica 6 - Funcoes 2/java_logic/Exercicio4_v1.java
import java.util.Scanner;

public class Exercicio4_v1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        System.out.println("Digite seu nome: ");
        String nome = sc.nextLine();
        System.out.println("Digite sua idade: ");
        int idade = sc.nextInt();

        System.out.println("Bem-vindo " + nome + ", você tem " + idade + " anos!");

        sc.close();
    }
}

File: Pratica 6 - Funcoes 2/cpp_logic/exercicio4_v1.cpp
#include <iostream>
using namespace std;

int main() {
    string nome;
    int idade;

    cout << "Digite o nome: ";
    cin >> nome;
    cout << "Digite a idade: ";
    cin >> idade;

    cout << "Bem-vindo " << nome << ", você tem " << idade << " anos!";

    return 0;
}